images/build/debian-base/variants.yaml

```diff
@@ -1,4 +1,4 @@
 variants:
   buster:
     CONFIG: 'buster'
-    IMAGE_VERSION: 'buster-v1.7.2'
+    IMAGE_VERSION: 'buster-v1.8.0'
```

Comment:
aside: IMO we should dedupe this versus the makefile and drop the dependencies.yaml entry. it's just friction.